scripts/build_ubuntu.sh

```diff
@@ -12,15 +12,16 @@ export TEST_WITH_OSTREE=0
 export TEST_WITH_TESTSUITE=0
 
 # build and copy aktualizr.deb and garage_deploy.deb to $TEST_INSTALL_DESTDIR
-"${GITREPO_ROOT}/scripts/test.sh"
+mkdir -p "$TEST_INSTALL_DESTDIR"
+"$GITREPO_ROOT/scripts/test.sh"
 
 # copy provisioning data and scripts
-cp -rf "${GITREPO_ROOT}/tests/test_data/prov_selfupdate" "${TEST_INSTALL_DESTDIR}"
-cp -rf "${GITREPO_ROOT}/tests/config/selfupdate.toml" "${TEST_INSTALL_DESTDIR}"
-cp -rf "${GITREPO_ROOT}/scripts/selfupdate_server.py" "${TEST_INSTALL_DESTDIR}"
-cp -rf "${GITREPO_ROOT}/tests/test_data/fake_root" "${TEST_INSTALL_DESTDIR}"
-cp -rf "${GITREPO_ROOT}/src/aktualizr_repo/run/create_repo.sh" "${TEST_INSTALL_DESTDIR}"
-cp -rf "${GITREPO_ROOT}/src/aktualizr_repo/run/serve_repo.py" "${TEST_INSTALL_DESTDIR}"
+cp -rf "$GITREPO_ROOT/tests/test_data/prov_selfupdate" "$TEST_INSTALL_DESTDIR"
+cp -rf "$GITREPO_ROOT/tests/config/selfupdate.toml" "$TEST_INSTALL_DESTDIR"
+cp -rf "$GITREPO_ROOT/scripts/selfupdate_server.py" "$TEST_INSTALL_DESTDIR"
+cp -rf "$GITREPO_ROOT/tests/test_data/fake_root" "$TEST_INSTALL_DESTDIR"
+cp -rf "$GITREPO_ROOT/src/aktualizr_repo/run/create_repo.sh" "$TEST_INSTALL_DESTDIR"
+cp -rf "$GITREPO_ROOT/src/aktualizr_repo/run/serve_repo.py" "$TEST_INSTALL_DESTDIR"
 
-git -C "${GITREPO_ROOT}" fetch --unshallow || true
-git -C "${GITREPO_ROOT}" describe > "${TEST_INSTALL_DESTDIR}/aktualizr-version"
+git -C "$GITREPO_ROOT" fetch --unshallow || true
+git -C "$GITREPO_ROOT" describe > "$TEST_INSTALL_DESTDIR/aktualizr-version"
```

Comment:
Any reason for removing the braces? I get that you probably don't need them here, but were they hurting something?